src/Components/Components/src/EventCallback.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.AspNetCore.Components
     /// <summary>
     /// A bound event handler delegate.
     /// </summary>
-    public readonly struct EventCallback
+    public readonly struct EventCallback : IEventCallback
```

Comment:
I also fixed #8336 in this PR because I was looking at it 😆

Reply:
Sounds good.

Is there any value in `EventCallback` being a `struct`? Are there any common use cases where it *won't* get boxed?

Reply:
Yes, in many of the common cases it's not boxed. Specifically places like `<input type="button" onclick="@OnClick" />`. Generally we have to box with a component, but not with an element.